src/ranged.cpp

```diff
@@ -247,7 +247,6 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
             // Don't do anything, especially don't call map::shoot as this would damage the vehicle
         } else {
             g->m.shoot( tp, proj, !no_item_damage && tp == target_arg );
-            has_momentum = proj.impact.total_damage() > 0;
```

Comment:
Thank you for catching this, I suspected that simple removing this line can ruine something important. I'll take a look for some better solution.